packages/tools/lib/monitor-postcss/index.js

```diff
@@ -0,0 +1,65 @@
+const chokidar = require("chokidar");
+const commandLineArgs = require("command-line-args");
+const psList = require("ps-list");
+const fkill = require("fkill");
+const { exec } = require("child_process");
+
+const options = commandLineArgs([
+	{ name: "srcFiles", type: String },
+	{ name: "packageName", type: String },
+]);
+const packageName = options.packageName;
+
+const restartPostcss = async () => {
+	let retryCount = 5;
+	let processes;
+
+	while (retryCount) {
+		try {
+			console.log("Getting processes list...");
+			processes = await psList();
+			retryCount = 0;
+		} catch (e) {
+			console.log("failed, retrying...");
+			retryCount--;
+		}
+	}
+
+	const process = processes.find(p => p.cmd.includes("postcss") && p.cmd.endsWith(`-w --packageName=${packageName}`));
```

Comment:
better make this `p.cmd.includes` which has the same effect but is less error prone in case someone adds more parameters to the postcss command

Reply:
I put `endsWith`because `@ui5/webcomponents` will also match `@ui5/webcomponents-fiori`.